src/docs/product/performance/transaction-summary.mdx

```diff
@@ -29,8 +29,9 @@ This graph shows the [average transaction duration](/product/performance/metrics
 
 On initial load, the table displays the slowest occurrences of the transaction along with the Event ID, User, Transaction Duration, and Timestamp of the event. Click on the Event ID to open the [event detail](/product/performance/event-detail/) for the transaction. Other sets of transactions you can browse through are below.
```

Comment:
We also need to change this sentence. `On initial load, the table displays slow occurrences of the transaction...`